scripts/project.js

```diff
@@ -68,4 +68,4 @@ document.addEventListener('deviceready', function() {
 document.addEventListener('backbutton', function() {
   // Return to the app's home screen
   window.location.replace('index.html');
-});
```

Comment:
what happened here?

Reply:
I- i dont remember editing that... i just edited line 33

Reply:
It says you removed and added the same line, I can’t see any visible difference

Reply:
The file used to end with a newline, you removed that.